packages/suite-desktop/src-electron/libs/menu.ts

```diff
@@ -17,7 +18,10 @@ const mainMenuTemplate: MenuItem[] = [
     // { role: 'fileMenu' }
     {
         label: 'File',
-        submenu: [isMac ? { role: 'close' } : { role: 'quit' }],
+        submenu: [
+            { label: 'Restart', click: restartApp },
```

Comment:
I guess it belongs more to `View` menu than `File`?

Reply:
I think `Restart` is more similar to `Close` than to `Reload`

Reply:
🤷